contracts/0.4.24/Lido.sol

```diff
@@ -686,7 +545,7 @@ contract Lido is Versioned, StETHPermit, AragonApp {
     }
 
     /**
-     * @dev Invokes a deposit call to the Staking Router contract and updates buffered counters
+     * @notice Invoke a deposit call to the Staking Router contract and update buffered counters
```

Comment:
next iteration idea: maybe we can extract all of the deposit logic outside of Lido?